libindy/src/services/pool/networker.rs

```diff
@@ -14,8 +12,8 @@ use utils::sequence;
 
 use super::time::Duration;
 
-use self::zmq::PollItem;
-use self::zmq::Socket as ZSocket;
+use super::zmq::PollItem;
```

Comment:
may it be used without `super::` here?

Reply:
Does not compile without the super::
I think this is because this is a module inside a module.

This PR as it is now removes all extern crates from all rust files exept when
- it is a lib.rs
- it is a mod.rs
- the rust file is 'included' in tests using 'path = "../../src/something"

If you find a way without the super:: I would be interested to learn how.

Reply:
ok, got it. I don't understand this compiler behavior but I can see the difference now (nested module in one file), thanks for explanation.